src/Message/Message.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import styles from './Message.module.scss'
```

Comment:
We have a convention that the imported `styles` should at the bottom of the imports

Reply:
Done.